components/device-mgt/org.wso2.carbon.device.mgt.core/pom.xml

```diff
@@ -222,6 +222,18 @@
             <artifactId>testng</artifactId>
         </dependency>
         <dependency>
+            <groupId>org.powermock</groupId>
+            <artifactId>powermock-module-testng</artifactId>
+            <version>1.7.0</version>
```

Comment:
Can you include this dependency in the root pom? And then import it here in the component pom? With that, we can make sure we reuse the same version of the dependency.

Reply:
ok